src/cases/containers/case-list/case-list.component.ts

```diff
@@ -7,6 +7,7 @@ import { Store, select } from '@ngrx/store';
 import { Observable, Subscription, combineLatest } from 'rxjs';
 import { Jurisdiction, CaseType, CaseState, SearchResultView, PaginationMetadata } from '@hmcts/ccd-case-ui-toolkit';
 import { FormGroup } from '@angular/forms';
+import { DefinitionsService } from '@hmcts/ccd-case-ui-toolkit/dist/shared/services/definitions/definitions.service';
```

Comment:
Directly referenced from CCD file. It should be @hmcts/ccd-case-ui-toolkit.

Reply:
Andy have approved this changes for now, until the ccd team has exported the service.